app/code/RLTSquare/Training/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rlt" translate="label" sortOrder="201">
+            <label>RLTSquare Training</label>
+        </tab>
+        <section id="rltsquare" translate="label" sortOrder="130" showInDefault="1" showInWebsite="1" showInStore="1">
+            <class>separator-top</class>
+            <label>Email Template</label>
+            <tab>rlt</tab>
+            <resource>RLTSquare_Training::email_config</resource>
+            <!-- for Email Configuration -->
+            <group id="training_email"
+                   translate="label"
+                   type="text"
+                   sortOrder="20"
+                   showInDefault="1"
+                   showInWebsite="1"
+                   showInStore="1">
+                <label>Email Configuration</label>
+                <field id="template"
```

Comment:
Copy-paste - why does this field have sortOrder="50". Assuming copy-paste? Please ensure fields sort orders is increments of 10... e.g. 10, 20, 30 ... this way you leave room to other module devs to add fields to your module in proper places, if need be.